skipscale/encrypt.py

```diff
@@ -52,8 +53,9 @@ async def encrypt(request):
     if not key:
         raise HTTPException(400, detail='Missing configuration')
 
-    image_prefix = request.app.state.config.encryption_url_prefix(tenant) + tenant + '/'
-    asset_prefix = request.app.state.config.encryption_url_prefix(tenant) + 'asset/' + tenant + '/'
+    config: Config = request.app.state.config
+    image_prefix = config.encryption_url_prefix(tenant) + tenant + '/'
+    asset_prefix = config.encryption_asset_url_prefix(tenant) + 'asset/' + tenant + '/'
```

Comment:
Hmm, if this has its own prefix key, do we still want the hardcoded asset prefix here?

Reply:
That would need magic in `planner` to detect  when the incoming request is using `asset_url_prefix`, and then asset requests would not work with other `app_path_prefixes`. Alternatively the asset-ness needs to be encoded in the encrypted url.

Keeping the additional component is the easiest if not the nicest solution.

Reply:
Ah, right. Guess we can live with /assets/v1/asset/ 👍